docs-2.0/3.ngql-guide/8.clauses-and-options/limit.md

```diff
@@ -1,38 +1,55 @@
 # LIMIT
 
-`LIMIT`子句限制输出结果的行数。
+`LIMIT`子句限制输出结果的行数。`LIMIT`在原生nGQL语句和openCypher兼容语句中的用法有所不同。
 
-- 在原生nGQL中，必须使用管道符（|），可以忽略偏移量。
+- 在原生nGQL中，使用管道符和不使用管道符有不同的语义，并且可以忽略偏移量。
 
 - 在openCypher方式中，不允许使用管道符，可以使用`SKIP`指明偏移量。
 
   !!! Note
 
         在原生nGQL或openCypher方式中使用`LIMIT`时，使用`ORDER BY`子句限制输出顺序非常重要，否则会输出一个不可预知的子集。
 
-## 原生nGQL语法
+!!! compatibility "历史版本兼容性"
 
-在原生nGQL中，`LIMIT`的工作原理与`SQL`相同，必须和管道符一起使用。`LIMIT`子句接收一个或两个参数。参数的值必须是非负整数。
+    Nebula Graph {{nebula.release}}中，`GO`和`LOOKUP`支持了新的`LIMIT`语法。部分`LIMIT`相关的算子支持计算下推。
```

Comment:
Lookup don't introduce new syntax.

Reply:
`| LIMIT` and `LIMIT` are different, aren't they? @Shylock-Hg

Reply:
Already change in https://github.com/vesoft-inc/nebula/pull/3171